Project-Euler/test/Problem017.test.js

```diff
@@ -0,0 +1,41 @@
+import { countNumberWordLength } from '../Problem017.js'
+
+describe('Number letter count', () => {
+  const textFixture = [
```

Comment:
Having both error & valid test cases here is somewhat dirty. Please use two calls to `each`. See https://github.com/TheAlgorithms/TypeScript/blob/master/CONTRIBUTING.md#writing-good-tests.